Lekce_7_HW/Car.cs

```diff
@@ -8,5 +8,38 @@ namespace Lekce_7_HW
 {
     internal class Car
     {
+        private string _brand;
+        private string _model;
+        private int _year;
+        private int _mileAge;
+
+        public Car(string brand, string model, int year, int mileAge)
+        {
+            _brand = brand;
+            _model = model;
+            _year = year;
+            _mileAge = mileAge;
+        }
+        public void Drive(int kilometres)
+        {
+            if (kilometres < 0)
+            {
+                Console.WriteLine("Ha! Někdo se nám tu snaží stočit tachometr!");
+                return;
+            }
+            else if (kilometres == 0) {
```

Comment:
Prvni if osetrujici přetočené  km je super! akorát ten `else` je tady navíc :)

```suggestion
            if (kilometres == 0) {
```

Jde o to, že jsi ošetřil vstup a vyhodnotil, že je něco špatně a rozhodl ses skončit pomocí `return` (to máš vše perfektně), není důvod, aby pak celá funkce "trpěla" tím, že je odsazená v rámci else bloku.

Nic to nekazí a je to funkčně správně, ale tohle je takový nepatrný syntax sugar, který ti výrazně zhezčí kód 😄